README.md

```diff
@@ -1,2 +1,127 @@
-# Hello-world
-hello Microvere Project
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [Understand the concept of version control](#about-project)
+  
+- [Follow GitHub flow.](#getting-started)
+  
+- [Open pull requests in GitHub](#authors)
+
+- [Write descriptive, easy to understand commit messages](#future-features)
+
+- [Write short, detailed, easy to understand descriptions of pull requests.](#contributing)
+
+- [Write a descriptive and easy to understand README in business English for every project submitted](#support)
+
+- [Demonstrate an ability to submit a project for code review.](#acknowledgements)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> In this project, I will set up a "Hello world" repository. My goal here is to master all of the tools and best practices you learned about in previous steps
+
+**[Hello Microverse]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **[.html file]**
+- **[.css file ]**
+- **[test.md]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Create
+
+-[GitHub repo]
+-[New branch]
+
+### Add
+-[.gitignore]
+-[node_modules].
+-[Descriptive README file].
+
+### Set up
+Linter for HTML AND CSS
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+@Chimane1987
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a thumbs up
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank Microverse for this opportunity to do this code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
### Great work implementing this project. I know you put in a lot of work to get it through. Kudos 👏 💪 

However, I have a few suggestions that will further improve your project for approval. 

- [ ]  First of all, your pull request should be from your `feature` branch into your `main` branch. This is the correct GitHub flow. [Please read more about this](https://docs.github.com/get-started/quickstart/github-flow). you may not be able to do much about this right now. ** I noticed that you closed the previous review, this is bad practice. Please, going forward do not open a new pull request after a review that was not approved, you are expected to fix all the issues and push to the same branch, this is good so the next reviewer can follow the history and give the best advice and suggestion/approval**

- [ ] I can see that you used the new Microverse README template which is great. For this template, kindly do the following when customizing it to suit this project:

* Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it. 🙏
* For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. 
* For the Client section, kindly include the technology you are using. 👍
* For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 👍
* For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:

      - Installation of Git
      - Installation of NPM
      - A working browser
      - Internet connection
      - A code editor like Visual Studio Code
      - Installation of Live Server plugging in Visual Studio Code
      - Any other tool you can think about

- For the Setup section, kindly add instructions for cloning your repository on a local machine 👍
- For the Install section, kindly add instructions for setting up linters used in this project
- For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.

- For the MIT section, kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE

https://choosealicense.com/licenses/mit/

#### Kindly implement all the changes listed above.